src/Model/Meta.php

```diff
@@ -77,6 +77,14 @@ public function getCreated()
         return $this->created;
     }
 
+    /**
+     * @param \DateTime $created
+     */
+    public function setCreated(\DateTime $created)
+    {
+        $this->created = $created;
+    }
+    
```

Comment:
Build have failed. php-cs-fixer is complaining about whitespace in this line

Reply:
That's strange, I copied the code from setLastModified() and only changed the names.
Could you please provide the exact message from php-cs-fixer?

Reply:
Click the red link below that says Travis failed, and find the php 7 build where php cs fixer is executed and you'll see the error message in the build log
https://travis-ci.org/tmilos/scim-schema/jobs/612621460